src/mmw/js/src/analyze/views.js

```diff
@@ -57,6 +57,33 @@ var ResultsView = Marionette.LayoutView.extend({
             newProjectUrl = '/project/new/' + modelPackageName,
             projectUrl = '/project';
 
+        if (modelPackageName === 'gwlfe') {
+            var analysisResults = JSON.parse(App.getAnalyzeCollection()
+                                                .findWhere({taskName: 'analyze'})
+                                                .get('result'));
+
+            var landResults = _.find(analysisResults, function(element) {
+                return element.name === 'land';
+            });
+
+            if (landResults) {
+                var landAreaResults = landResults.categories;
+
+                var landCoverTotal = _.reduce(_.filter(landAreaResults,
+                    function(category) {
+                        return category.type !== 'Open Water';
+                    }), function(sum, category) {
+                        return sum + category.area;
+                    }, 0);
+
+                if (landCoverTotal === 0) {
+                    window.alert('The selected Area of Interest doesn\'t ' +
```

Comment:
You wouldn't have to escape the apostrophe if you quoted the string in double-quotes. Just sayin'